response/writer.go

```diff
@@ -10,22 +10,22 @@ import (
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
-func WriteResponse(ctx context.Context, w http.ResponseWriter, babbageResponse *http.Response, req *http.Request, cfg *config.Config) {
+func WriteResponse(ctx context.Context, w http.ResponseWriter, clientResponse *http.Response, req *http.Request, cfg *config.Config) {
```

Comment:
Please remember to change all the instances of `clientResponse` in this file too.